cypress/defs.bzl

```diff
@@ -1 +1,95 @@
 "Public API re-exports"
+
+load("@aspect_rules_cypress//cypress/private:cypress_test.bzl", "cypress_test_lib")
+load("@aspect_rules_js//js:libs.bzl", "js_binary_lib")
+load("@aspect_bazel_lib//lib:directory_path.bzl", "directory_path")
+
+_cypress_test = rule(
+    doc = """Identical to js_test with the addition of the cypress toolchain made available.""",
+    attrs = cypress_test_lib.attrs,
+    implementation = cypress_test_lib.implementation,
+    test = True,
+    toolchains = js_binary_lib.toolchains + ["@aspect_rules_cypress//cypress:toolchain_type"],
+)
+
+def cypress_cli_test(name, cypress = "//:node_modules/cypress", **kwargs):
```

Comment:
hmm. How about calling this one `cypress_test` so it is the implied default?

From the looks of it the `cypress_module_test` is just a `cypress_test` in a bring your entry point & cypress node_modules.

Reply:
Done